webhooks/wh-general.md

```diff
@@ -7,7 +7,7 @@ You will receive events for all the properties you are integrated with, and can
 
 The message encapsulates all events occurring at the same time which belong to event types to which you have subscribed.
 For example, if you are interested in reservations you will subscribe to `ServiceOrderUpdated` events (a reservation is a type of Service Order)
-and you may receive multiple `ServiceOrderUpdated` events in one Webhook message. The message also contains the set of entities related to the events, as shown in the example below.
+and you may receive multiple `ServiceOrderUpdated` events in one Webhook message.
```

Comment:
Add something like: If a entity is added you may receive a event for both addition and update in one webhook message.

Reply:
@jirrick please check that my understanding is correct, but I would word it like this:

Note also that if an entity is added in Mews, the system may generate both an 'added' event and an 'updated' event in the same webhook message.

Reply:
I've added the message, it should be much clear now.